include/arch/arm/arch/machine/debug.h

```diff
@@ -206,12 +282,15 @@ static inline syscall_error_t Arch_decodeSetBreakpoint(tcb_t *t,
         }
     }
 
+#ifdef CONFIG_ARCH_AARCH32
     if (size == 8 && !byte8WatchpointsSupported()) {
```

Comment:
Cleaner to define a `byte8WatchpointsSupported()` macro for aarch64 that returns true.